include/mlir/Dialect/StandardOps/Ops.td

```diff
@@ -881,6 +881,14 @@ def ShlISOp : IntArithmeticOp<"shlis"> {
   let summary = "signed integer shift left";
 }
 
+def ShrISOp : IntArithmeticOp<"shris"> {
+  let summary = "signed integer shift right";
+}
```

Comment:
We should do a better job at providing extensive documentation here, as we intend to generate the documentation from this. See the doc for `sexti` above as an example

Reply:
I've written the doc for `sexti`. Glad you like it! :-) I'll write a similar paragraph also for the shifts.